### DISCUSSION TITLE
How to install a newer Ubuntu version?

### DISCUSSION THREAD

**Discussion by Gammer0909** [dnschneid/crouton#5036] [Q&A]
Following both the [Official Ubuntu Install Guide](https://ubuntu.com/tutorials/install-ubuntu-on-chromebook#1-overview), and the README, I can't figure out how to install a newer Ubuntu version (newer than 16.04). Can anyone help me figure out the `-n` parameter or how to install a newer version?

**Comment by DennisLfromGA** [comment]
@Gammer0909,

The Ubuntu Install Guide you linked above does not show the '**-r**' option that is needed to specify a release.
You would be better off following the crouton [README](https://github.com/dnschneid/crouton/blob/master/README.md) since it is a more complete guide.

It mentions the '**-r**' option's 'list' command here:
- https://github.com/dnschneid/crouton#hey-now-ubuntu-1604-is-pretty-old-im-young-and-hip

I'll show the output of that command below for your reference:

> Recognized debian releases:
    potato! woody! sarge! etch! lenny! squeeze! wheezy! jessie! stretch! buster
    bullseye bookworm sid
Recognized kali releases:
    moto! kali! sana! kali-rolling
Recognized ubuntu releases:
    warty! hoary! breezy! dapper! edgy! feisty! gutsy! hardy! intrepid! jaunty!
    karmic! lucid! maverick! natty! oneiric! precise! quantal! raring! saucy!
    trusty! utopic! vivid! wily! xenial yakkety! zesty! artful! bionic* cosmic!
    disco! eoan! focal* groovy! hirsute! impish* jammy* kinetic*
Releases marked with ! are upstream end-of-life, and should be avoided.
Releases marked with * are unsupported, but may work with some effort.

Once you see the list you can choose a release you want to install, I would stick with bionic, focal, or jammy.
Be aware that those releases are flagged with an asterik **\*** meaning they 'are unsupported, but may work with some effort.'

So to install the 'jammy' release you'd begin the command with: `sudo crouton -r jammy ...'

-**DennisLfromGA**

**Reply by Gammer0909** [reply]
I can't figure out the parameters or anything. Everything I try just brings up the help menu. (Which isn't seeming to be helpful...)

**Comment by DennisLfromGA** [comment]
@Gammer0909,

I'd suggest thoroughly reading the README first then if you still have problems tell us what you have tried including the complete commands you're using.

-**DennisLfromGA**
